front/src/applications/stdcm/formatStcmConf.ts

```diff
@@ -8,11 +8,12 @@ import { time2sec } from 'utils/timeManipulation';
 
 import { setFailure } from 'reducers/main';
 import { getPathfindingQuery } from 'common/Pathfinding/Pathfinding';
+import { InfraStateType } from 'reducers/infra';
```

Comment:
Could you also update the other type imports of the file please
```suggestion
import type { InfraStateType } from 'reducers/infra';
```

Reply:
Done